governance/votes/Executive vote - October 8, 2021.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Template - [Executive Vote] Offboarding Vaults, MKR Vesting, and Core Unit Budget Distribution - October 8, 2021
+summary: Offboards BAT-A, LRC-A, ZRX-A, UNIV2AAVEETH-A, and UNIV2LINKETH-A Vaults, establishes MKR vesting stream for RISK-001, and distributes the Core Unit Budget for CES-001.
+date: 2021-10-08T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Offboarding Vaults, MKR Vesting, and Core Unit Budget Distribution - October 8, 2021
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- BAT-A, LRC-A, ZRX-A, UNIV2AAVEETH-A, and UNIV2LINKETH-A Vaults will be offboarded as detailed below.
+- 700 MKR will be streamed to the [Risk Core Unit (RISK-001)](https://mips.makerdao.com/mips/details/MIP39c2SP2) as detailed below.
```

Comment:
I think that in effect this is just 700MKR on a certain cliff date. Because the cliff date = end_of_stream date.